src/lib/containers/DirectDownload.tsx

```diff
@@ -140,9 +141,11 @@ const DirectDownload: React.FunctionComponent<DirectFileDownloadProps> = props =
   const getIcon = () => {
     if (isExternalFile) {
       return (
-        <a rel="noreferrer" href={externalURL} target="_blank">
-          <Icon type="externallink" />
-        </a>
+        <button className={'btn-download-icon'}>
```

Comment:
https://sagebionetworks.jira.com/browse/PORTALS-1880
Output in the same button container so these icons are always aligned.